platform/barefoot/sonic-platform-modules-bfn-montara/debian/rules

```diff
@@ -6,7 +6,7 @@ SCRIPT_SRC := $(shell pwd)/scripts
 CONFIGS_SRC := $(shell pwd)/configs
 BUILD_DIR := $(shell pwd)/build
 WHEEL_BUILD_DIR := $(BUILD_DIR)/wheel
-PLUGINS_DIR := $(shell pwd)/plugins
+DEVICE_FILES_DIR := $(shell pwd)/device-files
```

Comment:
Please rename to `configs` which is more suitable for device configuration files

Reply:
please note we have configs/ dir which apparently has different meaning (/etc/ files)
change anyway?

Reply:
How about `/bin` vs `/usr/bin`? :)
Since this `configs` folder is inside concrete platform folder, it should be good to use it for platform/device configuration files storage. Note, inside pcied HLD pcie.yaml is mentioned as a configuration file. As for me, `device-files` is too generic name. It's like `some-files`.